jlee93115/freshly-produce
# Login Feature

## Problem:
On submit of correct username and password, matching user's information should be stored in vuex store and router should redirect to `Home`

## Solution:

### Summary:
Added login api to perform http request to server by passing json object containing inputted username and password.
On success, vuex store will be updated with username, user role, and user_id. Router will redirect to `Home` page.

### Steps to verify:
1. Confirm there is an existing user in `users` table in database
2. Confirm server side is up to date with `server-login` branch and running
3. Navigate to `http://<API_URL>:<API_HOST>/login/auth`
4. Login using correct credentials
5. Component should navigate to home page
